GliaWidgets/Layout/NSLayoutConstraint.Extensions.swift

```diff
@@ -0,0 +1,41 @@
+import Foundation
+import UIKit
+
+extension Array where Element == NSLayoutConstraint {
+    func activate() {
+        NSLayoutConstraint.activate(self)
+    }
+
+    func deactivate() {
+        NSLayoutConstraint.deactivate(self)
+    }
+
+    // swiftlint:disable operator_whitespace
+    static func +=(
```

Comment:
I think it may by sufficient to only use
```
NSLayoutConstraint.activate(
    [
        constarint1,
        constarint2,
        ..
    ]
)
```
and avoid using custom extension of Array. What do you think @dukhovnyi @kuldar-daniel ?

Reply:
I think this approach is a little bit better, just examples

```
var constraints = [NSLayoutConstraint](); defer { constraints.activate() }
constraints += buttonContainer.bottomAnchor.constraint(equalTo: bottomAnchor)
constraints += buttonContainer.leadingAnchor.constraint(equalTo: leadingAnchor)
constraints += buttonContainer.trailingAnchor.constraint(equalTo: trailingAnchor)
```

```
let bottomButtonContainer = buttonContainer.bottomAnchor.constraint(equalTo: bottomAnchor)
let leadingButtonContainer = buttonContainer.leadingAnchor.constraint(equalTo: leadingAnchor)
let trailingAnchorContainer = buttonContainer.trailingAnchor.constraint(equalTo: trailingAnchor)
```